### ISSUE TITLE
createTokenWithCard: UnpectedNativeTypeException with Android API 27

### ISSUE THREAD

**Issue by vbvxx** [tipsi/tipsi-stripe#478]
## The problem
I am using ```createTokenWithCard``` method. Everything is working with Android 9.0 (API 28). When I try with a simulator using Android 8.1 (API 27), I get the following crash:

<img width="401" alt="Screenshot 2019-05-21 at 16 19 16" src="https://user-images.githubusercontent.com/15009671/58104121-7eb40e00-7be4-11e9-878f-16e166599292.png">

## Environment
* `tipsi-stripe` version: 7.5.0 
* iOS or Android: Android
* OS version: 8.1 (build tools 27)
* React-Native version: 0.53.3
`com.google.android.gms:play-services-base` version: 12.0.1

## Links to logs and sources
```
stripe.setOptions({
  publishableKey: Config.STRIPE_PUBLISHABLE_KEY,
  androidPayMode: 'test',
});

const stripeToken = await stripe.createTokenWithCard({
    number,
    cvc,
    expMonth,
    expYear,
    name
  });
```

* android.compileSdkVersion 27
* android.buildToolsVersion 26.0.2
* android.defaultConfig.minSdkVersion 18
* android.defaultConfig.targetSdkVersion 27

**Comment by vbvxx** [comment]
I investigated a bit more with android studio debugger. It seems to me that the function ```createCard(cardData)``` is crashing but I couldn't get any additional information. However downgrading to version 7.4.0 fixed the issue 👍

**Comment by cybergrind** [comment]
@vbvxx that is very strange, because we're using 7.5.0 and our main testing device is android 5 and we're using createTokenWithCard.

would you mind switching it again to 7.5.0 and check after you have working 7.4.0 on your app?

**Comment by vbvxx** [comment]
@cybergrind sorry for coming back to you a bit late, I was wrong and this has nothing to do with your library. I was doing some date manipulation on expMonth / expYear the JSCore embedded with the RN version of my project for android OS < 28.
I realised that when I used again chrome debugger and realised the behaviour was inconsistent for the exact same apk (js code is executed with V8 when using chrome debugger). I guess this what happened when I assumed that downgrading the library had solved the issue.

Thanks for the great work 🙌
